packages/grafana-ui/src/themes/mixins.ts

```diff
@@ -82,3 +82,11 @@ export const getTooltipContainerStyles = (theme: GrafanaTheme2) => ({
   borderRadius: theme.shape.radius.default,
   zIndex: theme.zIndex.tooltip,
 });
+
+export const getExternalRadius = (radius: string, padding: number, borderWidth?: number): number => {
```

Comment:
I think we should add some JSDoc-style comments to these explaining which to use when. Like

```ts
/**
 * Calculates a border radius to use when ____
**/
export const getExternalRadius = blah blah
```


I'm not totally sure of it myself...  Is there an easy litmus test developers can apply to know when to use which?